Doc/library/sys.rst

```diff
@@ -81,14 +87,28 @@ always available.
 
    .. index:: single: auditing
 
-   Raise an auditing event with any active hooks. The event name is a string
-   identifying the event and its associated schema, which is the number and
-   types of arguments. The schema for a given event is considered public and
-   stable API and should not be modified between releases.
-
-   This function will raise the first exception raised by any hook. In general,
-   these errors should not be handled and should terminate the process as
-   quickly as possible.
+   Raise an auditing event and trigger any active auditing hooks.
+   `event` is a string identifying the event, and `args` may contain
+   optional arguments with more information about the event.  The
+   number and types of arguments for a given event are considered a
+   public and stable API and should not be modified between releases.
+
+   For example, one auditing event is named "os.chdir". This event has
+   one argument called `path` that will contain the requested new
+   working directory.
+
+   This function will call the existing auditing hooks in order,
+   passing the event name and arguments, and the first exception
+   raised by any hook will be re-raised. Native hooks added by
+   :c:func:`PySys_AddAuditHook` are called first, followed by hooks
+   added in the current interpreter.
+
+   In general, any exceptions raised by this function should not be
+   handled and should terminate the process as quickly as
+   possible. This allows hook implementations to decide how to respond
+   to any particular event. The typical responses will be to log the
```

Comment:
This sort of duplicates what is said earlier, but probably OK.

Reply:
There's no other section or module to link to, so the body of these two functions is basically going to be it for documentation of this auditing feature, so I'm trying to cover everything, but your point about duplication is good!  

I've reduced the discussion of sys.audit() a bit, leaving the hook order unspecified and taking out the PEP link.  The bulk of the documentation is now on sys.addaudithook(), which seems like the more complicated function and embodies more of the feature's API.